src/sdkV2.ts

```diff
@@ -2333,4 +2349,41 @@ export default class CegaEvmSDKV2 {
       },
     );
   }
+
+  async pendleUpdateAssetPrices(
+    timestamp: BigNumberish,
+    assetAddresses: { baseAsset: string; quoteAsset: string }[],
```

Comment:
these are `EvmAddresses` (consistent with pyth)
we can change it, but think it's best to keep these two similar as much as possible